From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sun, 21 Apr 2024 20:53:31 +0200
Subject: [PATCH] Disable tracker test

Our build containers break the sandbox and break the test.
---
 test/automated/displayless/meson.build | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/test/automated/displayless/meson.build b/test/automated/displayless/meson.build
index 4aaf36a9e15f..c105dbee3d61 100644
--- a/test/automated/displayless/meson.build
+++ b/test/automated/displayless/meson.build
@@ -46,9 +46,6 @@ tests = [
 ]
 
 tracker_tests = [
-  ['test-nautilus-search-engine-tracker', [
-    'test-nautilus-search-engine-tracker.c',
-  ]],
 ]
 
 foreach t: tests
